cmd/root.go

```diff
@@ -33,12 +33,41 @@ type APIClients struct {
 	ResourcesClient    resources.Client
 }
 
+type Command interface {
```

Comment:
We need an interface so the root command can keep track of its children that know if we showed a command or action's help. We only need this for non-resource commands because we can handle tracking the help in the root.